omaha/hammer.bat

```diff
@@ -77,7 +77,7 @@ set OMAHA_SIGNTOOL_SDK_DIR="%WindowsSdkVerBinPath%\x86"
 set PYTHONPATH=%OMAHA_PYTHON_DIR%
 
 :: Directory of Scons (http://www.scons.org/).
-set SCONS_DIR=C:\Python27\Lib\site-packages\scons-1.3.1
+set "SCONS_DIR=%OMAHA_PYTHON_DIR%\scons-1.3.1"
```

Comment:
Would it be possible to change the line 80 to 
`set "SCONS_DIR=%OMAHA_PYTHON_DIR%\Lib\site-packages\scons-1.3.1"`
so that the code builds like it used to before?

Thank you!

Reply:
I changed it because the original path appears to be incorrect. The new path is the one that scons is installed to when I perform the following:
```
python.exe scons-1.3.1\setup.py install
```

Reply:
It seems to me that the EXE installer sets it up under the site packages, whereas `setup.py` installs it under the Python dir.